actiontext/lib/action_text/fragment.rb

```diff
@@ -21,6 +21,8 @@ def from_html(html)
 
     attr_reader :source
 
+    delegate :deconstruct, to: "source.elements"
```

Comment:
As an alternative to making design decisions around the `#deconstruct` method, this change could instead forward all unknown methods from `Content` straight to `DocumentFragment`, through a chain of [delegate_missing_to](https://edgeapi.rubyonrails.org/classes/Module.html#method-i-delegate_missing_to) definitions.

That would mean that callers could be able to invoke `content.elements` directly. That would drastically broaden the public API of the class, but would remove the burden of making pattern matching interface considerations.